package.json

```diff
@@ -31,9 +31,10 @@
     "tool"
   ],
   "dependencies": {
-    "gulp-util": "^3.0.0",
     "buddy.js": "^0.8.0",
-    "through2": "^0.6.1",
-    "chalk": "*"
+    "chalk": "*",
+    "gulp-util": "^3.0.0",
+    "hook-stdio": "^1.0.0",
```

Comment:
This is the only line I added to this file, the rest was because I'm using npm@2 and this file was generated using npm@1. This is a module which does basically what the `hook_stdout` function was doing before, plus with some extra stuff.